requirements/test.txt

```diff
@@ -7,36 +7,36 @@
 amqp==1.4.9               # via kombu
 anyjson==0.3.3            # via kombu
 atomicwrites==1.3.0       # via pytest
-attrs==19.1.0             # via pytest
+attrs==19.3.0             # via pytest
 billiard==3.3.0.23        # via celery
 celery==3.1.26.post2
-configparser==3.7.4       # via importlib-metadata
-contextlib2==0.5.5        # via importlib-metadata
-coverage==4.5.3           # via pytest-cov
-ddt==1.2.1
-django-celery==3.3.0
+configparser==4.0.2       # via importlib-metadata
+contextlib2==0.6.0.post1  # via importlib-metadata
+coverage==4.5.4           # via pytest-cov
+ddt==1.2.2
+django-celery==3.3.1
 django-model-utils==3.0.0
 freezegun==0.3.12
 funcsigs==1.0.2           # via pytest
-future==0.17.1
-importlib-metadata==0.18  # via pluggy, pytest
+future==0.18.2
+importlib-metadata==1.1.0  # via pluggy, pytest
 jsonfield==2.0.2
 kombu==3.0.37             # via celery
-more-itertools==5.0.0     # via pytest
-packaging==19.0           # via pytest
-pathlib2==2.3.4           # via importlib-metadata, pytest, pytest-django
-pluggy==0.12.0            # via pytest
+more-itertools==5.0.0     # via pytest, zipp
+packaging==19.2           # via pytest
+pathlib2==2.3.5           # via importlib-metadata, pytest, pytest-django
+pluggy==0.13.1            # via pytest
 py==1.8.0                 # via pytest, pytest-catchlog
-pyparsing==2.4.0          # via packaging
+pyparsing==2.4.5          # via packaging
 pytest-catchlog==1.2.2
```

Comment:
`pytest-catchlog` has been merged into `pytest` and should be safe to remove from `requirements/test.in`.